controls/radgridview/events/column-resize-event.md

```diff
@@ -24,7 +24,7 @@ In this article we will discuss all RadGridView events related to **columns**:
 
 * [ColumnWidthChanged event](#columnwidthchanged-event)
```

Comment:
You are correct. The last two anchors should be just "ColumnWidthChanging" and "ColumnWidthChanged". Since the article is about "Column Events" I believe it's unnecessary to repeat the "event" word everywhere.